lib/ansible/modules/cloud/amazon/ec2_group.py

```diff
@@ -174,12 +174,13 @@
 import re
 import time
 from ansible.module_utils.basic import AnsibleModule
-from ansible.module_utils.ec2 import ec2_connect, ec2_argument_spec
+from ansible.module_utils import ec2 as ec2_utils
+from ansible.module_utils.ec2 import ec2_connect, ec2_argument_spec, get_aws_connection_info
+import ipaddress
 
 try:
-    import boto.ec2
-    from boto.ec2.securitygroup import SecurityGroup
-    from boto.exception import BotoServerError
+    import boto3
+    from botocore import exceptions
```

Comment:
`import botocore` will make this code a lot more consistent with most other boto3-based ansible modules